near-contract-standards/src/fungible_token/storage_impl.rs

```diff
@@ -68,6 +68,15 @@ impl StorageManagement for FungibleToken {
         self.internal_storage_balance_of(&account_id).unwrap()
     }
 
+    /**
+     * While storage_withdraw normally allows the caller to retrieve `available` balance, the basic
+     * Fungible Token implementation sets storage_balance_bounds.min == storage_balance_bounds.max,
+     * which means available balance will always be 0. So this implementation:
+     *
+     * * panics if `amount > 0`
+     * * never transfers Ⓝ to caller
+     * * returns a `storage_balance` struct iff `amount` is 0
```

Comment:
```suggestion
     * * returns a `storage_balance` struct if `amount` is 0
```

Reply:
"iff" meaning "[if and only iff](https://en.wikipedia.org/w/index.php?title=Iff&redirect=no)" 😃

Reply:
But I don't seem to be using that correctly so I'll change it 🙃